src/proto/primitive/primitive_msg_factory.cpp

```diff
@@ -11,10 +11,12 @@ std::unique_ptr<TbotsProto::Primitive> createMovePrimitive(
 {
     auto move_primitive_msg = std::make_unique<TbotsProto::Primitive>();
 
-    auto dest_msg        = createPointProto(Point(dest.x(), dest.y()));
+    TbotsProto::Path path_proto;
+    *(path_proto.add_point()) = *createPointProto(Point(dest.x(), dest.y()));
+    *(move_primitive_msg->mutable_move()->mutable_path()) = path_proto;
```

Comment:
Actually, I'm slightly confused. Wouldn't we want `createMovePrimitive` to take in a `Path` instead of a `Point dest`? @jonathanlew that was my understanding of it.

Reply:
since this will change so much with stp path planner re-architecture, it doesn't seem worth it to me to figure out what particular subset of those changes should be added to this PR

Reply:
Alright